CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+# Unreleased
+
+### Features
+- Add initial support for the RN New Architecture, backwards compatible RNSentry Turbo Module ([#2522](https://github.com/getsentry/sentry-react-native/pull/2522))
```

Comment:
```suggestion
### Features

- Add initial support for the RN New Architecture, backwards compatible RNSentry Turbo Module ([#2522](https://github.com/getsentry/sentry-react-native/pull/2522))
```

List the breaking changes with a breaking changes section, such as:
Min iOS supported version
Min RN engine supported version
etc...